src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3101,5 +3101,213 @@ internal virtual ImmutableArray<LabelSymbol> Labels
                 return ImmutableArray<LabelSymbol>.Empty;
             }
         }
+
+        /// <summary>
+        /// Perform a lookup for the specified method on the specified type.  Perform overload resolution
+        /// on the lookup results.
+        /// </summary>
+        /// <param name="patternType">Type to search.</param>
+        /// <param name="methodName">Method to search for.</param>
+        /// <param name="lookupResult">Passed in for reusability.</param>
+        /// <param name="warningsOnly">True if failures should result in warnings; false if they should result in errors.</param>
```

Comment:
>false if they should result in errors [](start = 83, length = 37)

I don't see any errors reported in here. I presume the caller is responsible for reporting errors; if I'm right this should probably say that instead.